sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client_connection_async.py

```diff
@@ -1105,7 +1105,7 @@ async def ReplaceTrigger(self, trigger_link, trigger, options=None, **kwargs):
         CosmosClientConnection.__ValidateResource(trigger)
         trigger = trigger.copy()
         if trigger.get("serverScript"):
-            trigger["body"] = str(trigger["serverScript"])
+            trigger["body"] = str(trigger.pop("serverScript", ""))
```

Comment:
Do we have any test for this change?

Reply:
the test would probably end up being an end to end test on every method for stored procedures, triggers, and udfs, which does not exist as of now

Reply:
yea, it would be good to add some tests to make sure in future, no regression happens. Will not block on this PR, but good to have a working item to track. Thanks